### PR TITLE
🔧 MAINTAIN: master --> main

### PR DESCRIPTION
**Pull request by rowanc1**
Also run prettier on various other files (github workflow and html).

See #4.

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -2,33 +2,31 @@ name: CI
...
-    branches: [ master ]
+    branches: [main]
 
 jobs:
-
   lint:
     runs-on: ubuntu-latest
 
     steps:
-    - uses: actions/checkout@v2
```
Comment (rowanc1) [inline .github/workflows/ci.yml:15]: No changes other than spaces on these - using the defaults of prettierrc in the repo.

File: .prettierignore [hunk 2]
```diff
@@ -0,0 +1 @@
+tests/fixtures/*.md
```
Comment (rowanc1) [inline .prettierignore:1]: The fixtures are a markdown file which prettier auto corrects, we don't want that.

File: docs/index.html [hunk 3]
```diff
@@ -1,11 +1,11 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset='utf-8'>
```
Comment (rowanc1) [inline docs/index.html:4]: Better html!

File: docs/index.html [hunk 4]
```diff
@@ -17,35 +17,42 @@ <h1>markdown-it-plugin-template</h1>
...
+          .use(window.markdownitExample)
+          .render(e.target.value)
         renderer.innerHTML = text
       }
 
       // trigger change handler for first render
-      const event = new Event("change");
-      inputText.dispatchEvent(event);
+      const event = new Event("change")
+      inputText.dispatchEvent(event)
```
Comment (rowanc1) [inline docs/index.html:55]: Follow our standards ... ! 💥

File: package-lock.json [hunk 5]
```diff
@@ -35,6 +35,9 @@
       "engines": {
         "node": ">=12",
         "npm": ">=6"
+      },
+      "peerDependencies": {
+        "markdown-it": "12.x"
```
Comment (rowanc1) [inline package-lock.json:40]: This was added to the package.json, but no `npm install` was run.

**Review by rowanc1 — COMMENTED** [review]
Overview of changes.

**Review by chrisjsewell — APPROVED** [review]

**Pull request merged**